contrib/platform/src/com/sun/jna/platform/win32/Winspool.java

```diff
@@ -571,6 +661,30 @@ boolean OpenPrinter(
             HANDLEByReference phPrinter,
             // _In_
             LPPRINTER_DEFAULTS pDefault);
+    
+    boolean GetForm(
```

Comment:
Please use the compact form of the signature (you are gaining nothing from spreading the signature over 10 lines). What would be nice is adding documentation.